### PR TITLE
Fixed snort2lua Version output error.

### PR DESCRIPTION
**Pull request by A-Pisani**
Solution to issue #213

### CODE REVIEW

File: tools/snort2lua/helpers/parse_cmd_line.cc [hunk 1]
```diff
@@ -280,6 +280,7 @@ static void set_ips_pattern(const char* /*key*/, const char* val)
 static void print_version(const char* /*key*/, const char* /*val*/)
```
Comment (snortadmin) [inline tools/snort2lua/helpers/parse_cmd_line.cc:280]: This needs to become noreturn. See eg print_binder_order.

Comment (A-Pisani) [reply tools/snort2lua/helpers/parse_cmd_line.cc]: Should be fine now.

**Comment by snortadmin** [comment]
Just noticed - there is no linefeed on the printf. Add a \n on the end and then squash your commits into one with a message in this form:

snort2lua: Fixed version output (issue #213)

Then I can pull that in through our CI process.

**Comment by A-Pisani** [comment]
There you go.

**Comment by snortadmin** [comment]
I ran that through our CI process and merged it internally. Look for your commit in our next release (2 weeks).

Thanks for the fix.

**Pull request closed without merging**